src/Reflection/ClassReflection.php

```diff
@@ -44,7 +44,8 @@ public function getDocBlock()
     /**
      * {@inheritDoc}
      *
-     * @param bool $includeDocComment
+     * @param  bool $includeDocComment
+     * @return int|false
```

Comment:
I would suggest trying `@inheritDoc` (no brackets) first.

Overall not opposed to this change, but to be attempted first.

Reply:
Wait... is this again because of Symfony's debug loader?

Reply:
There's no `@inheritDoc` data to read on the parent (internal) class unfortunately so this doesn't provide the needed info.

Reply:
Fair enough: I still smell the stench of `symfony/debug`, but I'll let this slip :P